rabbit-escape-render/src/rabbitescape/render/Polygon.java

```diff
@@ -0,0 +1,45 @@
+package rabbitescape.render;
+
+import java.util.Vector;
+
+import rabbitescape.engine.util.Util;
+
+public class Polygon
+{
+    public Vector<Integer> x, y;
```

Comment:
Making `x` and `y` public here makes it easy for an outsider to violate an invariant of this class, which is that x and y should have equal length (right?)